src/appendices/hed.md

```diff
@@ -178,11 +178,10 @@ repository on GitHub should be used to validate the study event annotations.
 }
 ```
 
-If you omit the `HEDVersion` field from the dataset description file,
-a warning will be generated and
-any present HED information will be validated using the latest version of the HED schema.
-This is bound to result in problems, and hence, it is strongly RECOMMENDED that the
-`HEDVersion` field be included when using HED in a BIDS dataset.
+An error will be generated by the BIDS validator if the `HEDVersion`
```

Comment:
>An error will be generated by the BIDS validator if the `HEDVersion`

replace with:

> An error will be generated by the BIDS validator if your dataset uses HED and the `HEDVersion` ....

Note: I confirmed this change has been merged into the bids validator (as of yesterday evening). The HED validator first looks for HED in the `events.tsv`, `events.json`, `participants.tsv`, and 1participants.json` files. It won't complain about the version unless HED is being used in any of these files.

Reply:
Approve with the above change....

Reply:
Thank you @VisLab !
Correction to the standard schema example and `HEDVersion` field clarification were made.

Reply:
Changed [here](https://github.com/bids-standard/bids-specification/pull/1592/commits/fabfe8443399d6fd07b9d48cb35aa44aaa8361e5)